app/Jobs/ServerConnectionCheckJob.php

```
@@ -95,6 +105,30 @@ public function handle()
         }
     }
 
+    private function checkHetznerStatus(): void
+    {
+        try {
+            $hetznerService = new \App\Services\HetznerService($this->server->cloudProviderToken->token);
```

PHPMD (2.15.0)

Missing class import via use statement (line '111', column '35'). (undefined)

(MissingImport)